### DISCUSSION TITLE
5.x 企业微信回调事件连续发送三次

### DISCUSSION THREAD

**Discussion by yinjisong** [w7corp/easywechat#2254] [General]
如题，企业微信解密出echostr原文，将原文当作Get请求的response，返回给公众平台这里，发现同一个事件发生了三次回调事件，如何在收到消息时及时向公众平台返回echostr原文

**Comment by overtrue** [comment]
什么版本？

**Reply by yinjisong** [reply]
5.7.1的版本，但是经过测试是有三次回调事件的，因为添加好友的时候可能需要做一些浪费时间的操作，然后就会出现这种情况

**Reply by overtrue** [reply]
把推送的消息时间内容记录对比一下看看是一样的还是不同的事件

**Reply by yinjisong** [reply]
嗯嗯，如果在遇到这种问题我再反馈吧，我把企业微信的客户回调做了判断，数据库里有就只是更新不会删除
